lntest/itest/lnd_test_list_group1_test.go

```diff
@@ -0,0 +1,78 @@
+// +build rpctest
+
+package itest
+
+var testCasesGroup1 = []*testCase{
```

Comment:
We could split programmatically instead, so we don't have to have four files for the groups. Also would make it easier to parametrize how parallel we want to run (I realize this is a bit more difficult to do but with predefined group names it'd be viable)

Reply:
The reason why I started with the manual split was because it seemed that some tests were more flaky depending on the order they were executed (as if there was some left-over state from previous runs). But it does make more sense to do the split programmatically and fix the flakes instead.